content/guides/testing.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Testing Integrations
+kind: guide
+listorder: 6
+autotocdepth: 2
+---
+<style>
+.func-p{margin-left:20px;}
+.func-code{margin-left:40px;}
+</style>
+
+### Overview
+
+Automated unit tests help ensure that your integration is working as designed and guard against regression.
+
+Though we don't require tests for each metric collected, we strongly encourage you to provide as much coverage as possible.
```

Comment:
@gmmeyer @truthbk @remh Can we come to consensus on what the minimum requirement for merging a check is?